src/global/notifications/notifications.android.ts

```diff
@@ -120,7 +118,7 @@ class LocalNotification extends NativeMobileComponent {
   private _indeterminate = false;
   private _ticker = '';
   private _vibrate = false;
-  private _priority = NotificationsAndroid.Priority.DEFAULT;
+  private _priority = Priority.DEFAULT;
```

Comment:
All these variables should be in init function for good measure. If we are sure none of those is used in constructor, this can stay as is.